### ISSUE TITLE
compose

### ISSUE THREAD

**Issue by semmel** [tarsana/functional#4]
```php
function compose() {
    return F\_apply('Tarsana\Functional\pipe', array_reverse(func_get_args()));
}
```
I find it sometimes more convenient to think in composing functions `(f · g)(x) = f(g(x))` instead of function pipes `x ➜ g ➜ f = f(g(x)` which have such an imperative touch.

**Comment by webNeat** [comment]
Thanks for the remark, I will add the `compose` function.

**Comment by webNeat** [comment]
Added here https://github.com/tarsana/functional/commit/687d2716230d6d94fa5190612d397c4eb96f018f

Will be included in version 2.2.0
